app/serializers/api/v1/package_serializer.rb

```diff
@@ -39,6 +39,14 @@ def sent_on__sql
       "stockit_sent_on"
     end
 
+    def on_hand_quantity
+      object.total_in_hand_quantity
+    end
+
+    def on_hand_quantity__sql
+      "(SELECT sum(quantity) FROM packages_inventories where packages_inventories.package_id = packages.id)"
```

Comment:
@swatijadhav I think it should be `packages_inventories.package_id = #{object.id}`

Reply:
Verified it on local, and it seems to be working fine. `object` is used mostly used in the related method definition. Ideally `__sql` should not `object` reference.

Reply:
sure, makes sense 👍